scripts/local_k8s/k3d.py

```
@@ -0,0 +1,66 @@
+    require_tools(("k3d",))
+    if not _cluster_exists(config.cluster_name):
+        raise LocalK8sError(f"k3d cluster {config.cluster_name!r} does not exist")
+    print(f"cluster: {config.cluster_name}")
+    print(f"ingress: http://127.0.0.1:{config.ingress_port}")
+
+
+def _cluster_exists(cluster_name: str) -> bool:
+    result = run("k3d", ["cluster", "list", "--output", "json"])
+    clusters = json.loads(result.stdout or "[]")
+    if not isinstance(clusters, list):
+        raise LocalK8sError("unexpected k3d cluster list JSON shape")
```

Ruff (0.15.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)